amqp/Go/README.md

```diff
@@ -0,0 +1,56 @@
+# IBM MQ Go AMQP 1.0 samples
+The Go samples are based on go-amqp (a Go AMQP 1.0 client implementation) as provided here: [AMQP 1.0 client module for Go](https://github.com/Azure/go-amqp)
+## Go Samples
+
```

Comment:
As the samples perform both put/get and pub/sub, a better name for them is amqpProducer.go and amqpConsumer.go